src/components/AppHeader/Header.js

```diff
@@ -1,17 +1,11 @@
 import React from "react";
 import "./HeaderStyle.scss";
-import {PersianLan} from "../../constants/Strings";
-import {AppBar, Toolbar, Typography} from "@material-ui/core";
+import logo from "../../logo.png";
```

Comment:
Irrelevant changes to this task. Send such these changes in a separate task, not "Map" please.